subgraphs/marketplace/package.json

```diff
@@ -14,10 +14,10 @@
     "deploy:arbitrum": "graph deploy --product hosted-service treasureproject/marketplace",
     "deploy:mainnet": "true",
     "deploy:staging": "graph deploy --product hosted-service treasureproject/marketplace-next",
-    "prepare:arbitrum-testnet": "true",
+    "prepare:arbitrum-testnet": "mustache ../../node_modules/@treasure/subgraph-config/src/arbitrum-testnet.json template.yaml > subgraph.yaml",
```

Comment:
A lot of the address configs used in the marketplace subgraph are missing for testnet

Reply:
Yup, just copied them over for now from production. Will eventually get them deployed properly on tesetnet.